dev-requirements.txt

```diff
@@ -15,6 +15,7 @@ graphviz~=0.20.0
 nuclio-sdk>=0.3.0
 isort~=5.7
 avro~=1.11
+mlflow~=2.5
```

Comment:
why here?

Reply:
needed for tests, where should it be?

Reply:
make tests install `mlrun[mlflow]`?

Reply:
why? we don't need it for run, user wants it he can install